Corrfunc/mocks/DDtheta_mocks.py

```diff
@@ -158,7 +158,7 @@ def DDtheta_mocks(autocorr, nthreads, binfile,
 
     max_cells_per_dim : integer, default is 100, typical values in [50-300]
         Controls the maximum number of cells per dimension. Total number of
-        cells can be up to (max_cells_per_dim)^3. Only increase if ``thetamax``
```

Comment:
The actual number of bins does depend on `thetamax`, but the maximum possible number of bins per dimension is limited by this `max_cells_per_dim` (set to 100 by default) -- will you please revert the replacement? That said,  this brings out a different typo in this line! The correct entry is that the total number of cells can be up to `(max_cells_per_dim)^2` since the binning is at most 2D (when `link_in_ra=True` -> binning in both RA and DEC).